docs/usage/types/list_types.md

```diff
@@ -1,60 +1,63 @@
 `list`
 : allows `list`, `tuple`, `set`, `frozenset`, `deque`, or generators and casts to a list;
-  see `typing.List` below for sub-type constraints
+  when a generic parameter is provided, the appropriate validation is applied to all items of the list
+
+`typing.List`
+: handled the same as `list` above
 
 `tuple`
 : allows `list`, `tuple`, `set`, `frozenset`, `deque`, or generators and casts to a tuple;
-  see `typing.Tuple` below for sub-type constraints
-
-`typing.List`
-: see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
+  when generic parameters are provided, the appropriate validation is applied to the respective items of the tuple
 
 `typing.Tuple`
-: see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
-
-`subclass of typing.NamedTuple`
-: Same as `tuple` but instantiates with the given namedtuple and validates fields since they are annotated.
-  See [Annotated Types](#annotated-types) below for more detail on parsing and validation
-
-`subclass of collections.namedtuple`
-: Same as `subclass of typing.NamedTuple` but all fields will have type `Any` since they are not annotated
+: handled the same as `tuple` above
 
-### Typing Iterables
+`deque`
+: allows `list`, `tuple`, `set`, `frozenset`, `deque`, or generators and casts to a deque;
+  when generic parameters are provided, the appropriate validation is applied to the respective items of the deque
 
-Pydantic uses standard library `typing` types as defined in PEP 484 to define complex objects.
+`typing.Deque`
```

Comment:
I moved `deque` here from `set` since I think it's a better place for it. (It's more similar to a list)